pom.xml

```diff
@@ -382,6 +382,11 @@
         <artifactId>graal-sdk</artifactId>
         <version>20.0.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.graalvm.nativeimage</groupId>
+        <artifactId>svm</artifactId>
+        <version>20.0.0</version>
```

Comment:
Is it safe to hardcode the version of the graal here?
The quarkus takes it from the env runtime variable. 
What will happen if someone will try to compile it in earlier versions?

Reply:
This lib is only required to gain access to the annotations needed to implement the substitution.  So long as the Graal native build respects those annotations there shouldn't be an issue.  I've tested this with a toolchain going back to 19.2.1 and it worked without issue so we should be fine.

Note that Quarkus only supports 19.3.1 so even with 19.2.1 we'd be talking about users writing their own apps for Graal and not using Quarkus.